kafka/main.go

```diff
@@ -185,13 +271,19 @@ func Emit(producer *kafka.Writer, message []byte, logger *zerolog.Logger) error
 
 func getDialer(logger *zerolog.Logger) *kafka.Dialer {
 	var dialer *kafka.Dialer
-	brokers = strings.Split(os.Getenv("KAFKA_BROKERS"), ",")
 	if os.Getenv("ENV") != "local" {
+		logger.Info().Msg("Generating TLSDialer")
 		tlsDialer, _, err := batgo_kafka.TLSDialer()
 		dialer = tlsDialer
 		if err != nil {
 			logger.Error().Msg(fmt.Sprintf("Failed to initialize TLS dialer: %e", err))
 		}
+	} else {
+		logger.Info().Msg("Generating Dialer")
+		dialer = &kafka.Dialer{
+			Timeout:   10 * time.Second,
```

Comment:
Sorry to ask, shouldn’t this be 20 seconds?

Given that the MaxWait is 20 seconds, the dialer might stop 10 seconds before. I’m not sure what would happen afterwards. Would the dialer start dialing again?

Reply:
[Timeout](https://github.com/segmentio/kafka-go/blob/e1f5945be0b2e606b6826e9e5b0ec0a9ce68bd93/dialer.go#L39) is how long the client will wait for a connection, where [MaxWait](https://github.com/segmentio/kafka-go/blob/382e96d3016e702bce4f51427f8fc7fc62f47b82/reader.go#L414) is how long a connected consumer will wait for a batch to be delivered over that connection. Honestly, both are probably too high and should be lowered but I don't think they'll conflict.

Reply:
Oh ok ok! Thank you 🙏